docs/GettingStarted.md

```diff
@@ -0,0 +1,80 @@
+# Getting started
+
+All you'll a [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and the Hail source code: either clone the [Hail repository](https://github.com/broadinstitute/hail) using [Git](https://git-scm.com/) with
+```
+git clone https://github.com/broadinstitute/hail.git
+```
+or download the source code directly from [here](https://github.com/broadinstitute/hail/archive/master.zip), unzip, and rename the enclosing `hail-master` directory to `hail`. The following commands are relative to this `hail` directory.
+
+## Building and running Hail
+
+Hail may be built to run locally with all cores on a single machine or to run on a Spark cluster. Running locally is useful for getting started, analyzing or experimenting with small datasets, and Hail development.
+
+### Building and running Hail locally
+The single command
+```
+./gradlew installDist
+```
+will populate `build/install` with an installation of Hail, including the executible `build/install/hail/bin/hail`. The initial build takes time as the build tool [Gradle](https://gradle.org/) installs and then downloads all Hail build dependencies, including the Scala compiler and Spark.
```

Comment:
I would say `build/install/hail` is an installation of Hail.  executable (spelling).  I'd just say "dependencies".